src/hw2.js

```diff
@@ -0,0 +1,104 @@
+// 1
+// Задача про обчислення різниці часу
+// Напишіть функцію яка буде буде приймати 3 параметри
+// — початкову дату (string)
+// — кінцеву дату (string)
+// — розмірність ('days', 'hours', 'minutes', seconds)
+// Функція повертатиме часовий період між цими датами згідно розмірності.
+// Також вкажіть значення по замовчуванню для всіх цих параметрів (на ваш вибір).
+// Функція має коректно працювати навіть якщо початкова дата пізніше ніж кінцева дата.
+
+// Приклади:
+// durationBetweenDates('02 Aug 1985', '03 Aug 1985', 'seconds') // поверне '86400 seconds'
+// durationBetweenDates('31 Jan 2022', '03 Feb 2021', 'days') // поверне '362 days'
+
+const DIMENSIONALITY = {
+  DAYS: "days",
+  HOURS: "hours",
+  MINUTES: "minutes",
+  SECONDS: "seconds",
+};
+
+const durationBetweenDates = (
+  startDate,
+  endDate,
```

Comment:
```suggestion
  startDate,
  endDate,
```

Тут у всіх параметрах мають бути значення за замовчуванням, як в тексті задачі вказано - "Також вкажіть значення по замовчуванню для всіх цих параметрів (на ваш вибір)."